code.ts

```diff
@@ -139,17 +145,33 @@ function secondsToInterval(seconds: number): string {
   return result;
 }
 
+
+/** 
+* Code that updates all timers on the Figma stage
+* will also send updates / messages to UI.html, so we can show timers there
+*/
+
 async function startTimer(node: TextNode, seconds: number, template: string, startsWithTimer: boolean) {
```

Comment:
the start timer function is getting hard to read. Maybe worth refactoring into separate functions

Reply:
Simplified the start timer function and also took some parts out in separate functions. Should be much easier to read now, what do you think?

Reply:
Its way better! There is more work to do but we can also address that in follow up PRs since some of the issues were not introduced by this PR